tz.cpp

```diff
@@ -3134,7 +3134,8 @@ current_zone()
         {
             std::ostringstream os;
             char message[128];
-            (void)strerror_r(errno, message, 128);
+            if (strerror_r(errno, message, 128) != 0)
+	      message[0] = '\0';
```

Comment:
Looks good but I smell a tab. ;-)
Also, the repeated 128 bugs me.  Want to slip `sizeof(message)` in there while you're in the neighborhood, and have to get rid of the tab anyway?